cmd/client.go

```diff
@@ -85,24 +86,40 @@ func (cli *Client) ShowJobSpec(c *clipkg.Context) error {
 	}
 	defer resp.Body.Close()
 	var job presenters.JobSpec
-	return cli.deserializeResponse(resp, &job)
+	return cli.renderResponse(resp, &job)
 }
 
 // GetJobSpecs returns all job specs.
 func (cli *Client) GetJobSpecs(c *clipkg.Context) error {
 	cfg := cli.Config
-	resp, err := utils.BasicAuthGet(
-		cfg.BasicAuthUsername,
-		cfg.BasicAuthPassword,
-		cfg.ClientNodeURL+"/v2/specs",
-	)
-	if err != nil {
-		return cli.errorOut(err)
-	}
-	defer resp.Body.Close()
 
-	var jobs []models.JobSpec
-	return cli.deserializeResponse(resp, &jobs)
+	requestURL := cfg.ClientNodeURL + "/v2/specs"
+	var allJobs []models.JobSpec
+	for {
+		resp, err := utils.BasicAuthGet(
```

Comment:
Looks like this loop performs a GET on every iteration. Is there no way to only perform one GET for the desired request?

Also, in the spirit of self documented code, could we but the body of this loop into a method with the method name describing the intent of the action? Makes it easier to read 🤓

Reply:
1. If you used a large enough size value, we do not have a limit.
2. Good idea.